.github/workflows/test.yml

```diff
@@ -95,6 +95,27 @@ jobs:
       run: pip install -e '.[llvm,testing]' --extra-index-url https://download.pytorch.org/whl/cpu
     - name: Run Pytest
       run: ENABLE_METHOD_CACHE=1 LLVM=1 python -m pytest -s -v -n=auto test/
+  
+  testclang:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+    runs-on: ${{ matrix.os }}
+    name: CLANG Tests ${{ matrix.os }} (w method cache)
+
+    steps:
+    - name: Checkout Code
+      uses: actions/checkout@v3
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v4
+      with:
+        python-version: 3.8
+    - name: Install Dependencies
+      run: pip install -e '.[testing]' --extra-index-url https://download.pytorch.org/whl/cpu
+    - name: Set env
+      run: printf "CI=1\nCLANG=1\nENABLE_METHOD_CACHE=1" >> $GITHUB_ENV
```

Comment:
ENABLE_METHOD_CACHE is the default, you don't need to explicitly turn it on (should be removed from LLVM).

I think `CI=1` is automatic too in CI

Reply:
Just checked github docs and `CI=true` by default so can remove it as well as ENABLE_METHOD_CACHE

Reply:
Yea, I'd merge a small PR cleaning it up, but no big deal.